### DISCUSSION TITLE
Filtering multiple categories

### DISCUSSION THREAD

**Discussion by johanvanwambeke** [bullhorn/career-portal#509] [General]
Hi,
I am bumping in a API limitation, it seems to be only returning 1 published category. And also only filtering on only 1 category.

```
"publishedCategory": {
    "id": 2000026,
    "name": "Boekhouder/Accountant"
 },
```

If you take a rather basic job description for IT it would have a few tags: .net, C#, Microsoft,...
And the job should show if any of these get filtered.

How do we enable this on the Angular career portal?

kind regards,
Johan

**Comment by pbearne** [comment]
I don't think you can do this in OSCP
If this becomes a deal breaker Matador Jobs can do this by default

I would love to talk to you about your job board and why you might want to upgrade.
I have created a short whitepaper to help explain -
https://matadorjobs.com/why-replace-oscp-with-matador-jobs/

If you are not the right person, can you connect me to them?
